### PR TITLE
Remove kickstart sources from TUI (4/5)

### PR DESCRIPTION
**Pull request by poncovka**
This is the fourth part of the code that will replace RPM sources in UI with
the DBus sources of the Payloads module. All parts have to be tested
and merged together.

Remove kickstart sources from TUI. Use the DBus sources instead.

**Depends on:** https://github.com/rhinstaller/anaconda/pull/2533

**DO NOT MERGE!**

### CODE REVIEW

**Comment by poncovka** [comment]
Rebased.

File: pyanaconda/ui/tui/spokes/installation_source.py [hunk 1]
```diff
@@ -376,14 +359,14 @@ def apply(self):
...
-            (self.data.method.server, self.data.method.dir) = self._nfs_server.split(":", 2)
+            host, path = self._nfs_server.split(":", 2)
         except ValueError as err:
             log.error("ValueError: %s", err)
             self._error = True
             return
 
         opts = self._nfs_opts or ""
-        self.set_source_nfs(opts)
+        self.set_source_nfs(opts=opts, host=host, path=path)
```
Comment (jkonecny12) [inline pyanaconda/ui/tui/spokes/installation_source.py:369]: This have to change to:
```suggestion
        self.set_source_nfs(server=host, directory=path, opts=opts)
```

https://github.com/rhinstaller/anaconda/pull/2533/commits/486b8e094a5a0044ad0e765cf9df8f64c53be8f2#diff-b4c6badd586849c76c60e56fc1a215e9R141

**Comment by poncovka** [comment]
Rebased.

File: pyanaconda/ui/tui/spokes/installation_source.py (hunk 1 above)
Comment (poncovka) [reply pyanaconda/ui/tui/spokes/installation_source.py]: Fixed.

**Comment by poncovka** [comment]
Jenkins, test this please.

**Comment by poncovka** [comment]
The tests are passing.

**Comment by poncovka** [comment]
Rebased.

**Comment by poncovka** [comment]
Tests pass when run locally.

**Pull request merged**